Tests/SwiftKueryORMTests/TestSave.swift

```diff
@@ -4,6 +4,38 @@ import XCTest
 import Foundation
 import KituraContracts
 
+/*
+  Function to extract the captured groups from a Regex match operation:
+  https://samwize.com/2016/07/21/how-to-capture-multiple-groups-in-a-regex-with-swift/
+**/
+extension String {
+    func capturedGroups(withRegex pattern: String) -> [String] {
+        var results = [String]()
+
+        var regex: NSRegularExpression
+        do {
+            regex = try NSRegularExpression(pattern: pattern, options: [])
+        } catch {
+            return results
+        }
+
+        let matches = regex.matches(in: self, options: [], range: NSRange(location:0, length: self.count))
+
+        guard let match = matches.first else { return results }
+
+        let lastRangeIndex = match.numberOfRanges - 1
+        guard lastRangeIndex >= 1 else { return results }
+
+        for i in 1...lastRangeIndex {
+            let capturedGroupIndex = match.range(at: i)
+            let matchedString = (self as NSString).substring(with: capturedGroupIndex)
```

Comment:
Apparently this doesn't work on Linux...

Reply:
Yes there's no bridging, you have to create the NSString using an initializer.